collector/excluded_progs.list

```diff
@@ -0,0 +1,140 @@
+^(
```

Comment:
On a somewhat more serious note, could we change this to be a straight up list of progs, then use something like `JOIN` to put them together `|` and simply add the `^(` and `)$` afterwards?
https://cmake.org/cmake/help/latest/command/string.html#join

Reply:
Yeah, why not.